index.src.html

```diff
@@ -17,96 +17,99 @@
 Text Macro: RP WebAuthn Relying Party
 Text Macro: RPS WebAuthn Relying Parties
 Text Macro: INFORMATIVE <em>This section is not normative.</em>
+Text Macro: WAC WebAuthn Client
 Ignored Vars: op, alg, type, algorithm
-Abstract: This specification defines an API that enables web pages to access WebAuthn compliant strong cryptographic credentials
-    through browser script. Conceptually, one or more credentials are stored on an authenticator, and each credential is scoped
-    to a single Relying Party. Authenticators are responsible for ensuring that no operation is performed without the user's
-    consent. The user agent mediates access to credentials in order to preserve user privacy. Authenticators use attestation
-    to provide cryptographic proof of their properties to the relying party. This specification also describes a functional
-    model of a WebAuthn compliant authenticator, including its signature and attestation functionality.
+Abstract: This specification defines a Web API enabling the creation and use of  strong, attested, cryptographic, scoped credentials
+    by web applications, for the purpose of strongly authenticating users. Conceptually, one or more credentials, scoped
+    to a given Relying Party, are created and stored on an authenticator. Authenticators are responsible for ensuring that no operation is performed without user 
+    consent. The user agent mediates access to credentials in order to preserve user privacy. Authenticators provide cryptographic proof of their properties to relying parties via attestation. This specification also describes the functional
```

Comment:
Suggest moving the user agent sentence up, to a point before you start describing the role of authenticators. That would allow the text to flow naturally into defining the model for authenticators. Otherwise you are ping-ponging from authenticators to user agents and back to authenticators.